packages/fast-components-react-base/src/typography/typography.tsx

```diff
@@ -23,6 +23,7 @@ class Typography extends Foundation<
 > {
     public static defaultProps: Partial<TypographyProps> = {
         tag: TypographyTag.p,
+        scalable: false,
```

Comment:
I prefer the name "scaled" - like you used in the class-name. To me, "scalable" implies I can take some action to scale this - not that the property itself causes scaling.